ios/FluentUI/Controls/Button.swift

```diff
@@ -101,6 +112,17 @@ open class Button: UIButton {
         }
     }
 
+    /// The button's image.
+    /// The image must be a template image.
+    /// For ButtonStyle.primaryFilled and ButtonStyle.primaryOutline, the image must be 24x24.
+    /// For ButtonStyle.secondaryOutline, the image must be 20x20.
+    /// For other styles, the image is not displayed.
+    @objc open var image: UIImage? {
+        didSet {
+            update()
+        }
+    }
```

Comment:
Can we add a guard to make sure it's a template image?

Reply:
Actually, this works even if the image is not a template image. There isn't really a way to know either as far as I can tell. I'll remove the comment. It should be expected that the image's tint color will be modified so the right kind of image must be provided.